package.json

```
@@ -3,10 +3,11 @@
   "private": true,
   "scripts": {
     "clean": "npx lerna clean",
-    "lint": "npm run lint --workspaces",
+    "lint": "npx eslint . && npm run lint --workspaces",
```

shouldn't need to npx this

```suggestion
    "lint": "eslint . && npm run lint --workspaces",
```